wallet/src/__test__/client/index.test.ts

```diff
@@ -0,0 +1,188 @@
+        });
+      });
+
+    const url = await topUpWallet({
+      userId: users[0],
+      country: "MX",
+      currency: "MXN",
+      amount: 1000.0,
+    });
+
+    expect(url).toBeTruthy(); // PAUSE the debugger here to get the URL and top up wallet manually
+  });
+
+  test("success: transfers an amount from a Rapyd ewallet balance to another Rapyd ewallet", async () => {
+    const requestCurrency = "MXN";
+    const requestAmount = 500.0;
+
+    const transferFromWallet = ({
+      senderUserId,
+      recipientUserId,
+      currency,
+      amount,
+    }: TransferFromWalletRequest.AsObject): Promise<TransferFromWalletReply.AsObject> =>
+      new Promise((resolve) => {
+        const request = new TransferFromWalletRequest();
+
+        request.setCurrency(currency);
+        request.setAmount(amount);
+        request.setSenderUserId(senderUserId);
+        request.setRecipientUserId(recipientUserId);
+
+        walletClient.transferFromWallet(request, (error, reply) => {
+          if (error) {
+            throw error;
+          }
+
+          resolve({
+            pendingTransactionId: reply.getPendingTransactionId(),
+            senderUserId: reply.getSenderUserId(),
+            recipientUserId: reply.getRecipientUserId(),
+          });
+        });
+      });
+
+    const { pendingTransactionId, senderUserId, recipientUserId } =
+      await transferFromWallet({
+        senderUserId: users[0],
+        recipientUserId: users[1],
+        currency: requestCurrency,
+        amount: requestAmount,
+      });
+
+    const setTransferFromWalletResponse = ({
+      senderUserId,
+      recipientUserId,
+      responseStatus,
+      pendingTransactionId,
+    }: SetTransferFromWalletResponseRequest.AsObject): Promise<SetTransferFromWalletResponseReply.AsObject> =>
+      new Promise((resolve) => {
+        const request = new SetTransferFromWalletResponseRequest();
+
+        request.setSenderUserId(senderUserId);
+        request.setRecipientUserId(recipientUserId);
+        request.setPendingTransactionId(pendingTransactionId);
+        request.setResponseStatus(responseStatus);
+
+        walletClient.setTransferFromWalletResponse(request, (error, reply) => {
+          if (error) {
+            throw error;
+          }
+
+          resolve({
+            amount: reply.getAmount(),
+            currency: reply.getCurrency(),
+            senderUserId: reply.getSenderUserId(),
+          });
+        });
+      });
+
+    const setTransferFromWalletResponseReply =
+      await setTransferFromWalletResponse({
+        senderUserId,
+        recipientUserId,
+        responseStatus: "accept",
+        pendingTransactionId,
+      });
+
+    expect(setTransferFromWalletResponseReply.amount).toEqual(requestAmount);
+    expect(setTransferFromWalletResponseReply.currency).toEqual(
+      requestCurrency
+    );
+    expect(setTransferFromWalletResponseReply.senderUserId).toEqual(users[0]);
+  });
```

Comment:
@Ktoxcon 
Successfully transferring from wallet to wallet. 

Let's start experimenting with the Bot and how to handle these requests. 

<img width="1440" alt="Screen Shot 2021-06-02 at 15 07 09" src="https://user-images.githubusercontent.com/4053518/120545523-ff996900-c3ab-11eb-8072-bbdc86c62976.png">

<img width="1440" alt="Screen Shot 2021-06-02 at 15 06 53" src="https://user-images.githubusercontent.com/4053518/120545519-fe683c00-c3ab-11eb-8851-809890394872.png">